dali/pipeline/operators/reader/reader_op.h

```diff
@@ -32,7 +32,7 @@ namespace dali {
  *
  * Operator runs an additional prefetch thread
  */
-template <typename Backend>
+template <typename Backend, typename LoadTarget, typename ParseTarget = LoadTarget>
```

Comment:
Could you update the class documentation with info, what is LoadTarget and ParseTarget?

Reply:
Done